src/blockchain/compact_filters/peer.rs

```diff
@@ -111,6 +110,7 @@ impl Mempool {
 
 /// A Bitcoin peer
 #[derive(Debug)]
+#[allow(clippy::dead_code)]
```

Comment:
Have to remove `clippy::`
```suggestion
#[allow(dead_code)]
```